### DISCUSSION TITLE
Canvas: trying to draw too large(202255872bytes) bitmap.

### DISCUSSION THREAD

**Discussion by liufeng18981858747** [coil-kt/coil#1191] [Q&A]
After updating version 2.0.0 today, the exception "canvas: trying to draw too large (202255872 bytes) bitmap" will be thrown when entering the list, I'm sure the gallery doesn't exceed the limit. Because I went back to 1.4.0, everything was normal.

**Comment by colinrtwhite** [comment]
This sounds like your `ImageView`'s height in the list is unbounded, which uses the original image's height in 2.x - which might be too large. You can bound the height by setting `size(Dimensions.Original, Dimension(1920)).scale(Scale.FILL)`.

**Reply by liufeng18981858747** [reply]
I'm sure that's not the reason, because after I go back to 1.4.0, everything returns to normal, and there is no infinite limit to the height of ImageView
